guides/common/modules/proc_installing-the-discovery-service.adoc

```diff
@@ -21,21 +23,15 @@ ifdef::foreman-el,katello[]
 --foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/
 ----
 endif::[]
-. Restart {Project} services:
-+
+. Install `foreman-discovery-image`:
```

Comment:
Like above, this is limited to satellite (and possibly orcharhino).

Reply:
From the earlier lines, I have considered including `Orcharhino` alongside. Addressed :+1: